tests/modules/sightings/resources/utils.py

```diff
@@ -195,3 +195,26 @@ def send_sage_identification_response(
             response, expected_status_code, {'status', 'message'}
         )
     return response
+
+
+def create_sighting_and_individual(
```

Comment:
This seems like a useful util that should be moved to individual utils or sighting utils. There is already a similar one in the individual utils so that seems like the best place.

Reply:
Actually following on from what I said in slack on asset groups, I now think this is wrong and we should be creating utils in asset_group that create the required stuff as an asset group.